src/pybind/mgr/dashboard/controllers/iscsi.py

```diff
@@ -64,7 +64,27 @@ def status(self):
     @Endpoint()
     @ReadPermission
     def settings(self):
-        return IscsiClient.instance().get_settings()
+        def to_boolean(value):
+            value = str(value).lower()
+            if value in ['1', 'true', 'yes', 'y']:
+                return True
```

Comment:
I'd really surprised if we haven't had anything like this already in the code. [`strtobool`](https://docs.python.org/3.5/distutils/apiref.html#distutils.util.strtobool)? But keep in mind it returns 0 or 1, so you still need to feed a `bool()`.

Reply:
Fixed, thanks.